app/vmui/packages/vmui/src/components/ExploreAlerts/ItemHeader/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { useAppState } from "../../../state/common/StateContext";
+import Tooltip from "../../Main/Tooltip/Tooltip";
+import classNames from "classnames";
+import { useNavigate } from "react-router-dom";
+import Badges, { BadgeColor } from "../Badges";
+import {
+  LinkIcon,
+  GroupIcon,
+  AlertIcon,
+  AlertingRuleIcon,
+  RecordingRuleIcon,
+  DetailsIcon,
+} from "../../Main/Icons";
+import Button from "../../Main/Button/Button";
+
+interface ItemHeaderControlsProps {
+  entity: string;
+  type?: string;
+  groupId: string;
+  states?: Record<string, number>;
+  id?: string;
+  name: string;
+  onClose?: () => void;
+}
+
+const ItemHeader: FC<ItemHeaderControlsProps> = ({ name, id, groupId, entity, type, states, onClose }) => {
+  const { isMobile } = useDeviceDetect();
+  const { serverUrl } = useAppState();
+  const navigate = useNavigate();
+  const copyToClipboard = useCopyToClipboard();
+
+  const openItemLink = () => {
+    navigate({
+      pathname: "/rules",
+      search:   `group_id=${groupId}&${entity}_id=${id}`,
+    });
+  };
+
+  const copyLink = async () => {
+    let link = `${serverUrl}/#/rules?group_id=${groupId}`;
+    if (type) link = `${link}&${entity}_id=${id}`;
+    await copyToClipboard(link, `Link to ${entity} has been copied`);
+  };
+
+  const headerClasses = classNames({
+    "vm-explore-alerts-item-header": true,
+    "vm-explore-alerts-item-header_mobile": isMobile,
+  });
+
+  const renderIcon = () => {
+    switch(entity) {
+      case "alert":
+        return (
+          <Tooltip title="Alert">
+            <AlertIcon />
+          </Tooltip>
+        );
+      case "group":
+        return (
+          <Tooltip title="Group">
+            <GroupIcon />
+          </Tooltip>
+        );
+      default:
+        switch(type) {
+          case "alerting":
+            return (
+              <Tooltip title="Alerting rule">
+                <AlertingRuleIcon />
+              </Tooltip>
+            );
+          default:
+            return (
+              <Tooltip title="Recording rule">
+                <RecordingRuleIcon />
+              </Tooltip>
+            );
+        }
+    }
+  };
+
+  return (
+    <div
+      className={headerClasses}
+      id={`rule-${id}`}
+    >
+      <div className="vm-explore-alerts-item-header__title">
+        {renderIcon()}
+        <div className="vm-explore-alerts-item-header__name">{name}</div>
+      </div>
+      <Badges
+        items={Object.fromEntries(Object.entries(states || {}).map(([name, value]) => [name, {
+          color: name.toLowerCase().replace(" ", "-") as BadgeColor,
+          value: value == 1 ? 0 : value,
+        }]))}
+      >
+        {onClose ? (
+          <Button
+            className="vm-back-button"
+            size="small"
+            variant="outlined"
+            color="gray"
+            startIcon={<LinkIcon />}
+            onClick={copyLink}
+          >
+            <span className="vm-button-text">Copy Link</span>
```

Comment:
The button now seems copying incorrect URL.
<img width="758" height="435" alt="image" src="https://github.com/user-attachments/assets/58158f24-aa18-4661-afcc-9b65b84cd2ec" />

The copy gets the following link `http://localhost:8428/#/rules?group_id=15701237453163353695&alert_id=7888404066739011861`

This link suppose to open this alert again when clicked, but it does not:
<img width="872" height="460" alt="image" src="https://github.com/user-attachments/assets/e448112b-bd83-42ed-98f5-b3a777172eef" />